src/omero_zarr/raw_pixels.py

```diff
@@ -83,17 +66,42 @@ def planeGen() -> np.ndarray:
         longest = longest // 2
         level_count += 1
 
-    paths = add_raw_image(
-        planes=planes,
-        size_z=size_z,
-        size_c=size_c,
-        size_t=size_t,
-        d_type=d_type,
-        parent=parent,
-        level_count=level_count,
-        cache_dir=cache_dir,
-        cache_file_name_func=get_cache_filename,
-    )
+    # if big image...
+    if image.requiresPixelsPyramid():
+        paths = add_big_image(image, parent, level_count)
```

Comment:
Ah - you mean *always* use a tile-based approach, even if it's not a big image?
One question then would be: should the chunk-size match the whole plane (as it currently does for non-big images) or would it be better to have max plane of e.g. 1024 or 512 (which would be a ~breaking change)?

Actually, that reminds me that I should update to get a preferred tile-size from the rawPixelsStore, instead of hard-coding it at 512 x 512....